docs/source/pages/supported_technologies/index.rst

```diff
@@ -5,27 +5,55 @@
 Supported Technologies
 ======================
 
+-----------
+Build Tools
+-----------
+
+Macaron is able to detect the build and deployment scripts for the following build tools and package managers while analyzing the CI configurations,
+such as GitHub Actions workflows.
+
+* Maven
+* Gradle
+* Pip
+* Poetry
+* npm
+* Yarn
+* Go
+* Docker container
```

Comment:
```suggestion
* Docker
```
Because I think Docker is a tool while Docker container in this case is not.

Reply:
See commit [4afc607](https://github.com/oracle/macaron/pull/539/commits/4afc607a121890332d21ab39dac01fca39e2d24f)